Keda.Cosmosdb.Scaler/src/Program.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Net;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.AspNetCore.Server.Kestrel.Core;
+using Microsoft.Extensions.Hosting;
+using Microsoft.Extensions.Logging;
+
+namespace Keda.Cosmosdb.Scaler
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            CreateHostBuilder(args).Build().Run();
+        }
+
+        public static IHostBuilder CreateHostBuilder(string[] args) =>
+            Host.CreateDefaultBuilder(args)
+                .ConfigureLogging(logging =>
+                {
+                    logging.ClearProviders();
+                    logging.AddConsole();
+                })
+            .ConfigureWebHostDefaults(webBuilder =>
+            {
+                webBuilder.ConfigureKestrel(options =>
+                {
+                    options.Listen(IPAddress.Any, 4050, o => o.Protocols = HttpProtocols.Http2);
```

Comment:
- localhost instead for `IpAddress.Any`?
- Is hard coding port to `4050` ok? What if another service is using that port?

Reply:
I think it needs to be `IPAddress.Any` to accept external traffic, which will be the case since KEDA connects to it from another pod/node and there isn't a reverse proxy running in front of it, right? 

For the port, I think it's fine to assume default of `4050` or whatever you pick, but also might wanna respect a `PORT` env var if defined. Since each external scaler runs in its own pod/container, port conflicts shouldn't matter because the container runtime allow mapping the actual port the application is listening on.